content/components/action-menu.mdx

```diff
@@ -8,40 +8,69 @@ import {Box, Button, Heading, Link} from '@primer/react'
 
 <img
   width="960"
-  alt="Action menu opened with a label for the trigger (button), overlay (menu), and the internal action list."
-  src="https://user-images.githubusercontent.com/18661030/226487158-06c6cc38-65de-438c-8028-373862fa0dfe.png"
+  alt=""
+  src="https://user-images.githubusercontent.com/980622/233374605-cee72894-fc5e-46f4-a1ea-e82aa26084ef.png"
 />
 
-## Options
+The action menu is an extremely versatile component that is not restricted to a single type of action list items at a time. It can be used to create a combination of single select, multi-select, commands, and submenus.
+
+By using dividers and section titles, a hierarchy can be established within the action menu, making it easier to navigate and understand. This allows users to quickly find and access the actions they need, while also providing a clear and organized structure to the menu.
 
 <img
-  src="https://user-images.githubusercontent.com/18661030/226486461-c43f75c5-d9ed-415f-907c-9d2dbf71e3a2.png"
+  src="https://user-images.githubusercontent.com/980622/234302061-4a6c820f-5467-4648-b70e-04d16ff2b5a5.png"
   width="960"
   alt=""
 />
-<Caption>
-  The trigger button can be a standard or icon button. For single select menus that display the selection in the
-  trigger, a label must be persist either internally or externally.
-</Caption>
+
+As the action menu is a highly adaptable component, it's crucial to keep the interactions with individual items limited. The following interactions are recommended:
+
+- `enter`, `click`, or `tap`: These actions trigger an action or navigate into a submenu, based on the selected item.
+- `arrow left`, `arrow up`, `arrow down`, `arrow right`: These actions can be used to navigate the menu, allowing the user to move between items and submenus with ease.
+
+Menus are a commonly used feature across multiple platforms, and both [Apple](https://developer.apple.com/design/human-interface-guidelines/components/menus-and-actions/menus) and [Microsoft](<https://learn.microsoft.com/en-us/previous-versions/windows/desktop/bb226797(v=vs.85)>) have detailed documentation on their menus. To create a pleasant user experience, it's important to follow their expected behavior closely.
+
+## Anatomy
 
 <img
-  src="https://user-images.githubusercontent.com/18661030/226486463-893231bb-8f57-4255-83a4-7d468b2be5ed.png"
   width="960"
-  alt=""
+  alt="Action menu opened with a label for the trigger (button), overlay (menu), and the internal action list."
+  src="https://user-images.githubusercontent.com/980622/234074196-f67bf01e-29b6-4be1-a6fc-47fc532f536b.png"
 />
-<Caption>Menus can contain simple actions and links, single select items, or multi select items.</Caption>
 
-## Behavior
+## Content
+
+### Open
+
+Action menus can be opened through a [button](/components/button), [icon button](/components/icon-button) or by right clicking list items.
```

Comment:
Maybe we should be more specific about what kind of list items open a menu when right clicking.

```suggestion
Action menus can be opened through a [button](/components/button), [icon button](/components/icon-button) or by right clicking list items that have a custom context menu associated with them.
```

I can help refine the wording if you're on board with this idea.